app/views/users/_auth_info.html.haml

```diff
@@ -0,0 +1,3 @@
+%span.auth-info
+  = t 'user.signed_in_as', user: current_user.email
+  = link_to t('user.logout').downcase, destroy_user_session_path
```

Comment:
@vlasiak, I'm not a fan of using Ruby code for displaying/styling purpose. Instead I suggest to use css class in case there is a need to apply text decorations.